src/components/bridge/ExternalBridge.vue

```diff
@@ -1,38 +1,33 @@
 <template>
   <div class="bridgeDash">
-    <q-card class="swapCard">
-      <div class="row justify-center">
-        <div class="text-h6 text-center q-pa-sm">
-            <div>
-                Attention
-            </div>
-          <div class="row justify-center text-body2">
-            If you wish to bridge between Wax, Telos and EOS, click the link below.
-          </div>
-          <div class="q-pb-sm">
-            <q-btn :href="checkTestNet()" type="a" class="q-ml-sm bridgeButton text-body">
-              Bridge 
-              <i role="presentation" class="fas fa-external-link-alt q-pl-sm"/>
-            </q-btn>
-          </div>
-        </div>
+    <q-card class="swapCard column items-center q-pa-md" flat>
+      <div class="text-h6 q-mb-sm">Attention</div>
+      <div class="text-body2 q-mb-sm text-center">
+        If you wish to bridge between Telos, EOS and WAX, click the link below.
+      </div>
+      <div class="">
+        <q-btn
+          :href="checkTestNet()"
+          type="a"
+          target="_blank"
+          class="q-ml-sm bridgeButton text-body"
+        >
+          EOSIO Bridge
+          <i role="presentation" target="_blank" class="fas fa-external-link-alt q-pl-sm" />
+        </q-btn>
       </div>
     </q-card>
   </div>
 </template>
 
 <script>
-  export default {
-    methods: {
-      checkTestNet(){
-        if (process.env.TESTNET === "true"){
-          return process.env.TSWAPS_BRIDGE_TEST;
-        } else{
-          return process.env.TSWAPS_BRIDGE;
-        }
+export default {
+  methods: {
+    checkTestNet() {
+        return process.env.TSWAPS_BRIDGE;
```

Comment:
Move this to the `data()` property of this component. We can call it `TSWAPS_BRIDGE` in all caps to show that it is an environment variable. See [MainLayout.vue](https://github.com/Yknot-BCS/tswaps-ui/blob/3a31ac2b1d0b80da25729fb1364dad452abc7fc3/src/layouts/MainLayout.vue#L208) for an example